### DISCUSSION TITLE
Infer org and repo from URL

### DISCUSSION THREAD

**Discussion by kblok** [community/community#9136] [Other Feature Feedback, Questions, & Ideas]
Switching to the web editor on GitHub is easy. You just change the `.com` (https://github.com/microsoft/playwright) to `.dev` (https://github.dev/microsoft/playwright) and you are already editing the repo.
In the same way, it would be great if I could change `https://github.com/microsoft` to `https://cs.github.com/microsoft` and I'm ready to search on the `microsoft` org. Or `https://github.com/microsoft/playwright` or `https://cs.github.com/microsoft/playwright` and search inside the `playwright`'s repo.

**Comment by gorzell** [comment]
Hi @kblok. I am pretty sure that is the existing behavior for full repo paths. i.e. https://cs.github.com/torvalds/linux.  It's a bit subtle, but you can see that the scope is set next to the search box.

![image](https://user-images.githubusercontent.com/1234453/146780084-9b919f93-bfc3-4d19-ad2d-afd0afd03ce0.png)

This is not the case for orgs at the moment, but it is certainly some that we can look at adding.

**Reply by kblok** [reply]
Ah that's cool. My main use-case was at org level. On an org with over 100 repos, look for a piece of code (or error message).
